docs/user/using_the_client.md

```diff
@@ -6,6 +6,12 @@ title: Using the Client
 Rucio provides several commands for the end-user. See [executables](bin/rucio.md).
 The command line client is called ``rucio``.
 
+Please note that these commands are not meant to be included in python scripts. 
+For such applications, we encourage using the [python client](pathname:///html/site/client.html). 
```

Comment:
```suggestion
Note: for Python scripts, we encourage using the [Python client](pathname:///html/site/client.html) instead of the commands listed here.
```